tests/nn/data_parallel/test_fsdp_optimizer_utils.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+import functools
+from time import time
+
+from parameterized import parameterized
+import torch
+from torch.optim import SGD, Adadelta, Adam  # type: ignore
```

Comment:
we usually don't have typing in test files. so "type: ignore" is not needed?

Reply:
I was getting "torch.optim has no Attribute Adadelta" from mypy without this, using 

```bash
mypy --ignore-missing-imports --scripts-are-modules --pretty .
```
from `fs_test`.

Reply:
I see. magic mypy. I thought it would skip the whole file since there isn't any type annotation in it.